R/req-options.R

```diff
@@ -197,13 +198,13 @@ verbose_message <- function(prefix, x) {
   cli::cat_line(prefix, lines)
 }
 
-verbose_header <- function(prefix, x, redact = TRUE) {
+verbose_header <- function(prefix, x, redact = TRUE, to_redact = NULL) {
```

Comment:
`to_redact` is needed as `line` is parsed into a header and therefore doesn't have the `redact` attribute.